<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DebuggerUIClass</class>
 <widget class="QMainWindow" name="DebuggerUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>627</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DebuggerUI</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionColoring"/>
   <addaction name="actionClone_Explorer"/>
   <addaction name="actionlogs"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>627</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad_configuration"/>
    <addaction name="actionSave_configuration"/>
    <addaction name="actionLoad_Dump"/>
    <addaction name="actionSave_Dump"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionClone_Explorer">
   <property name="icon">
    <iconset resource="DebuggerUI.qrc">
     <normaloff>:/DebuggerUI/resources/cloneView.png</normaloff>:/DebuggerUI/resources/cloneView.png</iconset>
   </property>
   <property name="text">
    <string>Clone Explorer</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionlogs">
   <property name="icon">
    <iconset resource="DebuggerUI.qrc">
     <normaloff>:/DebuggerUI/resources/log.png</normaloff>:/DebuggerUI/resources/log.png</iconset>
   </property>
   <property name="text">
    <string>logs</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionColoring">
   <property name="icon">
    <iconset resource="DebuggerUI.qrc">
     <normaloff>:/DebuggerUI/resources/coloring.png</normaloff>:/DebuggerUI/resources/coloring.png</iconset>
   </property>
   <property name="text">
    <string>Coloring</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionLoad_configuration">
   <property name="text">
    <string>Load configuration</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave_configuration">
   <property name="text">
    <string>Save configuration</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionLoad_Dump">
   <property name="text">
    <string>Load Dump</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionSave_Dump">
   <property name="text">
    <string>Save Dump</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+D</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="DebuggerUI.qrc"/>
 </resources>
 <connections/>
</ui>
